include/mbedtls/x509_crt.h

```diff
@@ -651,4 +651,9 @@ int mbedtls_x509write_crt_pem( mbedtls_x509write_cert *ctx, unsigned char *buf,
 }
 #endif
 
+#ifndef MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_CERTIFICATES
+/* The test infrastructure requires a positive define */
+#define MBEDTLS_X509__DEFAULT_FORBID_SHA1
+#endif
```

Comment:
I was reviewing some PRs and found out that the infrastructure can indeed check if a macro is not defined. Take a look at https://github.com/ARMmbed/mbedtls/blob/development/tests/suites/test_suite_entropy.data#L56.